### ISSUE TITLE
Profile Chart missing data?

### ISSUE THREAD

**Issue by 7yl4r** [luislizcano/3D-wetlands-app#5]
Is the profile chart display of the LC classes not showing all the data?
For the selection below I was expecting to see a land cover class for each pixel.
Instead I only see a total of four land cover points.
The transect defintely spans more than four pixels.

![image](https://user-images.githubusercontent.com/1051390/118421821-dd0b0e80-b68f-11eb-83e7-34bff421d629.png)

**Comment by luislizcano** [comment]
Yes, I had to change some settings to plot a point every ~50 meters along the transect to avoid exceeding memory limits for longer transects.

**Comment by 7yl4r** [comment]
Sounds like a reasonable choice. Maybe let's add a note about that to the docs?

Let's leave this issue open for now though. A potential enhancement would be to add a setting and/or autoscaling for the point density.
